osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -71,8 +71,12 @@ private void load()
 
         protected virtual void ApplyHitAnimations()
         {
-            JudgementBody.ScaleTo(0.9f);
-            JudgementBody.ScaleTo(1, 500, Easing.OutElastic);
+            if (bodyDrawable.Drawable is LegacySkinExtensions.SkinnableTextureAnimation) { }
```

Comment:
Just a passing thought, what if this line was replaced with checking if ``bodyDrawable.Drawable`` is at all animated. Would remove the legacy notation and would not replace the current default implementation.

Reply:
Not viable in the long run, if the drawable-based skinning system happens. People will have animations randomly not work here in that case.

Reply:
If people have an animated drawable they probably want to disable scaling here anyway. If these scaling effects aren't disabled, the animation might look off (that's the reason I opened #10195). Also, the line doesn't disable all animations, it disables only those two scale transformations.

Reply:
This doesn't make the proposal sound any better, sadly. What if they *do* want to resolve all animations? Then it's back to square one.

The provided direction will ensure this should not be a problem in any case going forward.